src/Stratis.Bitcoin.Features.Interop/InteropPoller.cs

```diff
@@ -1270,7 +1276,8 @@ private void AddComponentStats(StringBuilder benchLog)
 
             foreach (ConversionRequest request in requests)
             {
-                benchLog.AppendLine($"Mint To: {request.DestinationChain} Address: {request.DestinationAddress.Substring(0, 10)}... Id: {request.RequestId} Status: {request.RequestStatus} Amount: {new Money(request.Amount)} Ext Hash: {request.ExternalChainTxHash}");
+                // TODO: Improve the readability of the amount field
+                benchLog.AppendLine($"Mint To: {request.DestinationChain}] Address: {request.DestinationAddress.Substring(0, 10)}... Id: {request.RequestId} Status: {request.RequestStatus} Amount: {request.Amount} Ext Hash: {request.ExternalChainTxHash}");
```

Comment:
Hmmm can we look at improvng this before we merge? Otherwise testing might become a bit convoluted having to convert etc? It is difficult?